prefect_docker/scripts/configuration/odissei_settings.toml

```diff
@@ -0,0 +1,55 @@
+HR={"ALIAS"="hr", "METADATA_DIRECTORY"="@format {this.HR_METADATA_DIRECTORY}", "SOURCE_DATAVERSE_URL"="@format {this.DATAVERSE_NL_URL}", "SOURCE_DATAVERSE_API_KEY"="@format {this.DATAVERSE_NL_API_KEY}", "DESTINATION_DATAVERSE_URL"="@format {this.ODISSEI_URL}", "DESTINATION_DATAVERSE_API_KEY"="@format {this.ODISSEI_API_KEY}"}
+LEIDEN={"ALIAS"="leidenuniversity", "METADATA_DIRECTORY"="@format {this.LEIDEN_METADATA_DIRECTORY}", "SOURCE_DATAVERSE_URL"="@format {this.DATAVERSE_NL_URL}", "SOURCE_DATAVERSE_API_KEY"="@format {this.DATAVERSE_NL_API_KEY}", "DESTINATION_DATAVERSE_URL"="@format {this.ODISSEI_URL}", "DESTINATION_DATAVERSE_API_KEY"="@format {this.ODISSEI_API_KEY}"}
+MAASTRICHT={"ALIAS"="maastricht", "METADATA_DIRECTORY"="@format {this.MAASTRICHT_METADATA_DIRECTORY}", "SOURCE_DATAVERSE_URL"="@format {this.DATAVERSE_NL_URL}", "SOURCE_DATAVERSE_API_KEY"="@format {this.DATAVERSE_NL_API_KEY}", "DESTINATION_DATAVERSE_URL"="@format {this.ODISSEI_URL}", "DESTINATION_DATAVERSE_API_KEY"="@format {this.ODISSEI_API_KEY}"}
+TILBURG={"ALIAS"="tiu", "METADATA_DIRECTORY"="@format {this.TILBURG_METADATA_DIRECTORY}", "SOURCE_DATAVERSE_URL"="@format {this.DATAVERSE_NL_URL}", "SOURCE_DATAVERSE_API_KEY"="@format {this.DATAVERSE_NL_API_KEY}", "DESTINATION_DATAVERSE_URL"="@format {this.ODISSEI_URL}", "DESTINATION_DATAVERSE_API_KEY"="@format {this.ODISSEI_API_KEY}"}
+TRIMBOS={"ALIAS"="trimbos", "METADATA_DIRECTORY"="@format {this.TRIMBOS_METADATA_DIRECTORY}", "SOURCE_DATAVERSE_URL"="@format {this.DATAVERSE_NL_URL}", "SOURCE_DATAVERSE_API_KEY"="@format {this.DATAVERSE_NL_API_KEY}", "DESTINATION_DATAVERSE_URL"="@format {this.ODISSEI_URL}", "DESTINATION_DATAVERSE_API_KEY"="@format {this.ODISSEI_API_KEY}"}
+UMCU={"ALIAS"="UMCU", "METADATA_DIRECTORY"="@format {this.UMCU_METADATA_DIRECTORY}", "SOURCE_DATAVERSE_URL"="@format {this.DATAVERSE_NL_URL}", "SOURCE_DATAVERSE_API_KEY"="@format {this.DATAVERSE_NL_API_KEY}", "DESTINATION_DATAVERSE_URL"="@format {this.ODISSEI_URL}", "DESTINATION_DATAVERSE_API_KEY"="@format {this.ODISSEI_API_KEY}"}
+UTRECHT={"ALIAS"="UU", "METADATA_DIRECTORY"="@format {this.UTRECHT_METADATA_DIRECTORY}", "SOURCE_DATAVERSE_URL"="@format {this.DATAVERSE_NL_URL}", "SOURCE_DATAVERSE_API_KEY"="@format {this.DATAVERSE_NL_API_KEY}", "DESTINATION_DATAVERSE_URL"="@format {this.ODISSEI_URL}", "DESTINATION_DATAVERSE_API_KEY"="@format {this.ODISSEI_API_KEY}"}
+VU={"ALIAS"="vuamsterdam", "METADATA_DIRECTORY"="@format {this.VU_METADATA_DIRECTORY}", "SOURCE_DATAVERSE_URL"="@format {this.DATAVERSE_NL_URL}", "SOURCE_DATAVERSE_API_KEY"="@format {this.DATAVERSE_NL_API_KEY}", "DESTINATION_DATAVERSE_URL"="@format {this.ODISSEI_URL}", "DESTINATION_DATAVERSE_API_KEY"="@format {this.ODISSEI_API_KEY}"}
```

Comment:
Ik heb nog iets gevonden om de redundancy van de laatste vier keys van al deze dicts (SOURCE_DATAVERSE_URL,  SOURCE_DATAVERSE_API_KEY, DESTINATION_DATAVERSE_URL en DESTINATION_DATAVERSE_API_KEY) niet te hoeven herhalen:

```
[dataverse_nl]
SOURCE_DATAVERSE_API_KEY =  "@format{this.DATAVERSE_NL_API_KEY}"
DESTINATION_DATAVERSE_URL = "https://destination.dataverse.com"
SOURCE_DATAVERSE_URL = "https://source.dataverse.com"

[UMCU]
ALIAS="UMCU"
METADATA_DIRECTORY="@format {this.UMCU_METADATA_DIRECTORY}"
[dataverse_nl]

[SUBVERSE]
ALIAS=""
METADATA_DIRECTORY=""
[dataverse_nl]
```

Ik weet niet of de syntax 100% klopt, maar op deze manier is het misschien iets leesbaarder en hebben we wat minder herhaling. Het is alleen een opmaak change dus we kunnen dit ook later op een andere ticket fixen.

Reply:
Dynaconf gebruikt [tables](https://toml.io/en/v1.0.0#table) zoals `[development]` en `[production]` om settings binnen een environment te bundelen, [layered environments](https://www.dynaconf.com/settings_files/#layered-environments-on-files) Daarom gebruik ik [inline tables](https://toml.io/en/v1.0.0#inline-table) om alle settings van een workflow te bundelen. (Van de toml specificatie mag je geen enters gebruiken in een inline table. Erg handig... 🤦)

Volgens mij kunnen we jouw voorstel niet gebruiken. Dit zal meerdere environments aanmaken zoals `[dataverse_nl]` en `[UMCU]`. En we kunnen maar in een environment tegelijkertijd zitten.